runtime/autoload/health/nvim.vim

```diff
@@ -3,9 +3,20 @@ let s:bad_responses = [
             \ 'unable to parse python response',
             \ 'unable to parse',
             \ 'unable to get pypi response',
-            \ 'unable to get neovim executable'
+            \ 'unable to get neovim executable',
+            \ 'unable to find neovim version'
```

Comment:
I think it's good now, The problem was that when I changed things to using the list, as Shougo mentioned I should, for system calls, it broke several calls, but I couldn't figure out where. So while I was debugging, I just added this, and then figured it'd probably be more helpful to just leave it. I can try and remove all of it, but it seems like in the end that just takes more work. I will make the change you mentioned above,